bashbot.sh

```diff
@@ -10,6 +10,8 @@
 # This file is public domain in the USA and all free countries.
 # Elsewhere, consider it to be WTFPLv2. (wtfpl.net/txt/copying)
 
+TMPDIR="./tmp-bot-bash"
```

Comment:
I set TEMPDIR by default to a subdir of telegram-bot-bash. If you don't like this change it to whatever dir you like.